changelogger/config/routes.rb

```diff
@@ -1,4 +1,4 @@
 Rails.application.routes.draw do
-  root to: 'home#show'
-  resources :webhooks, only: :create
+  get  '/', to: 'home#show'
```

Comment:
just decided to use the less railsy way of declaring this since I thought it would be clearer but don't feel super strongly 🤷‍♀️